### PR TITLE
Added source-map-loader webpack plugin to load source maps in packages/*

### PR DESCRIPTION
**Pull request by paninaro**
Source maps haven't been loading from packages other than packages/gui for a while now. This adds the `source-map-loader` webpack plugin to load those source maps.

### CODE REVIEW

File: packages/gui/webpack.react.babel.ts [hunk 1]
```diff
@@ -132,6 +132,10 @@ export default {
   ].filter(Boolean),
   module: {
     rules: [{
+      test: /\.js$/,
+      enforce: "pre",
+      use: ["source-map-loader"],
+    }, {
       test: /node_modules[\/\\](iconv-lite)[\/\\].+/,
```
Comment (paninaro) [inline packages/gui/webpack.react.babel.ts:139]: This is the most relevant part of the PR

File: packages/gui/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "scripts": {
         "start": "npm run electron",
         "dev:electron": "cross-env NODE_ENV=development webpack --config webpack.electron.babel.ts --mode development && electron .",
-        "dev:react": "cross-env NODE_ENV=development NODE_OPTIONS=--max_old_space_size=4096 webpack serve --config webpack.react.babel.ts --mode development",
+        "dev:react": "cross-env NODE_ENV=development NODE_OPTIONS=--max_old_space_size=8192 webpack serve --config webpack.react.babel.ts --mode development",
```
Comment (paninaro) [inline packages/gui/package.json:15]: I ran into an out of memory issue, so I doubled this value.

**Comment by ChiaMineJP** [comment]
Known Issue:
- There are lines where break point cannot be set.
 
https://user-images.githubusercontent.com/84098616/160229523-7b0be56a-22f5-4dac-b8e7-d326f703fe11.mov

**Comment by paninaro** [comment]
@ChiaMineJP Thanks for your fixes!

**Review by seeden — APPROVED** [review]
LGTM

**Pull request merged**